### PR TITLE
Reworking of Paraview and TTK build: catalyst adaptor working in WFA

### PR DESCRIPTION
**Pull request by rupertnash**
This branch removes TTK's patch of paraview and just build that normally via super build. PV now includes GDAL and numpy. I've also updated the build of WFA to enable catalyst by default.

### CODE REVIEW

File: packages/paraview/env.sh [hunk 1]
```diff
@@ -13,23 +13,33 @@ if [ -z "$INSTALLER_PARAVIEW_ENV_SH" ]; then
...
+
     installer_init
 
-    declare -A cmake_vars
     cmake_vars[BUILD_TESTING]=OFF
     cmake_vars[PARAVIEW_BUILD_SHARED_LIBS]=ON
     cmake_vars[PARAVIEW_BUILD_EDITION]=CATALYST_RENDERING
     cmake_vars[paraview_SOURCE_SELECTION]=$version
     cmake_vars[ENABLE_boost]=ON
+    cmake_vars[USE_SYSTEM_boost]=ON
```
Comment (vollia) [inline packages/paraview/env.sh:30]: why using the system boost ?

File: packages/ttk/env.sh [hunk 2]
```diff
@@ -12,22 +12,10 @@ if [ -z "$INSTALLER_TTK_ENV_SH" ]; then
     build_dir_name=build-$version
 
     # TTK needs a patched version of Paraview
-    pv_version=5.8.1
-    pv_source_dir_name=ParaView-v$pv_version
+    pv_version=5.9.0
```
Comment (vollia) [inline packages/ttk/env.sh:15]: update here too.

File: packages/paraview/env.sh [hunk 3]
```diff
@@ -13,23 +13,33 @@ if [ -z "$INSTALLER_PARAVIEW_ENV_SH" ]; then
     name=paraview
     version=5.9.0
```
Comment (vollia) [inline packages/paraview/env.sh:14]: we should use 5.9.1, or even master / git

File: packages/paraview/env.sh [hunk 4]
```diff
@@ -13,23 +13,33 @@ if [ -z "$INSTALLER_PARAVIEW_ENV_SH" ]; then
...
-    declare -A cmake_vars
     cmake_vars[BUILD_TESTING]=OFF
     cmake_vars[PARAVIEW_BUILD_SHARED_LIBS]=ON
     cmake_vars[PARAVIEW_BUILD_EDITION]=CATALYST_RENDERING
     cmake_vars[paraview_SOURCE_SELECTION]=$version
     cmake_vars[ENABLE_boost]=ON
+    cmake_vars[USE_SYSTEM_boost]=ON
+    cmake_vars[ENABLE_numpy]=ON
+    cmake_vars[USE_SYSTEM_numpy]=ON
     cmake_vars[ENABLE_protobuf]=ON
```
Comment (vollia) [inline packages/paraview/env.sh:33]: Where is TTK now ? in #5 I update this part to use the one provided by the superbuild

**Review by vollia — APPROVED** [review]
I do not understand where is built the TTK module now, is this PR dependant on #5 ?

File: packages/paraview/env.sh (hunk 1 above)
Comment (rupertnash) [reply packages/paraview/env.sh:30]: I'd equally ask why not? Boost is available as a system module so means the install is a little faster and uses less disk

**Comment by rupertnash** [comment]
TTK is built as a separate module, as before. Except now this module does not need to build its own, patched, version of ParaView

**Comment by rupertnash** [comment]
I should add that this does not depend on #5, but this is now installed and working on Cirrus at least.

**Pull request merged**